src/components/ProjectsList/ProjectsList.js

```diff
@@ -1,14 +1,36 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { List, LinkItem } from "../List/List";
 import "./ProjectsList.css";
-const projects = [ {_id: "234iuhowef9803rh", title: "Big Ben" }, {_id: "584pduablf9571qt", title: "London Bridge" } ]
-export default function ProjectList(props) {
+import projectService from "../../Services/ProjectService";
+
+export default function ProjectList() {
+
+  const [projects, setProjects] = useState([]);
+
+  
+  const getProject = async() => {
```

Comment:
Leave a comment next to this, stating that this needs to be changed to "getProjectByUserId"

Reply:
Negative, i think you have looked into the wrong spot, ProjectsList is functioning properly and this does not require any id.

The method i do is to **FIND ALL** the project inside the project collections thats why no specify is required